tests/controllers/test_collection_exercise_controller.py

```diff
@@ -122,23 +118,10 @@ def test_get_collection_exercises_with_events_and_samples_by_survey_id(self, moc
                 ],
             }
         ]
-        sample_summary = {
```

Comment:
Sample summary has been moved into collection exercise in the form of 
Integer sampleSize;
List<SampleLink> sampleLinks;
Do we want to check that is the case where valid in a test?

Reply:
The adding of the sample size and sample links to the collection exercise seems to be done on the collection exercise service side of things and is tested out quite thoroughly there. I don't think these two variables are ever referenced to on r-ops outside of HTML files.

Reply:
I think we still need to add the values to the collection exercise we create in the tests


```
collection_exercises = [
  {
      "id": collection_exercise_id,
      "name": "201601",
      "scheduledExecutionDateTime": "2017-05-15T00:00:00Z",
      "state": "PUBLISHED",
      "exerciseRef": "000000",
      "events": [
          {
              "collectionExerciseId": "14fb3e68-4dca-46db-bf49-04b84e07e77c",
              "eventStatus": "PROCESSED",
              "id": "b4a36392-a21f-485b-9dc4-d151a8fcd565",
              "tag": "mps",
              "timestamp": "2018-03-16T00:00:00.000Z",
          }
      ],
     "sampleSize": "8",
     "sampleLink": "UUID"
  }
]
```

Kind of like this, since this is what collection exercise would be returning, so we can test on the proper data.

The issue now is that all of the test data we have in the rOps tests is in an old format as it expects, but I'm not sure how we want to deal with that, probably question for the larger team/tech session